### PR TITLE
Vite!

### PR DESCRIPTION
**Pull request by reesericci**
This PR adds vite, using the dev server to serve the API as a middleware

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
 node_modules
 dist
-uploads/*
```
Comment (jaspermayone) [inline .gitignore:4]: This needs to stay!

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,24 @@
-.env
```
Comment (jaspermayone) [inline .gitignore:1]: Keep this!

**Review by jaspermayone — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (reesericci) [reply .gitignore:4]: oop yikes, i think npm create overwrote this

File: .gitignore (hunk 2 above)
Comment (reesericci) [reply .gitignore:1]: yes lmao

**Review by jaspermayone — APPROVED** [review]

**Pull request merged**